crates/proof-of-sql/src/sql/proof_plans/filter_exec_test.rs

```diff
@@ -192,7 +192,7 @@ fn we_can_get_an_empty_result_from_a_basic_filter_on_an_empty_table_using_first_
             ColumnType::Decimal75(Precision::new(75).unwrap(), 0),
         ),
     ];
-    let first_round_builder = &mut FirstRoundBuilder::new();
+    let first_round_builder = &mut FirstRoundBuilder::new(0);
```

Comment:
You should start from the input table length instead of 0

Reply:
these tables are empty so the lengths should be zero right